src/main/java/picard/sam/SplitSamByNumberOfReads.java

```diff
@@ -69,7 +70,9 @@
     @Argument(doc = "Input SAM/BAM file to split", shortName = StandardOptionDefinitions.INPUT_SHORT_NAME)
     public File INPUT;
 
-    @Argument(shortName = "N_READS", doc = "Split to have approximately N reads per output file.", mutex = {"SPLIT_TO_N_FILES"})
+    @Argument(shortName = "N_READS", doc = "Split to have approximately N reads per output file. The actual number of " +
+            "reads per output file will vary by no more than the number of output files times one minus the maximum " +
```

Comment:
I think you swapped "times" and "minus one" leading to a description seems wrong.